cypress/integration/login.js

```diff
@@ -1,29 +1,31 @@
+import LoginPage from '../objects/loginPage';
+import TimeLoggingPage from '../objects/timeLoggingPage';
+import PageComponents from '../objects/pageComponents';
+
+const loginPage = new LoginPage();
+const timeLoggingPage = new TimeLoggingPage();
+const pageComponents = new PageComponents();
+
 describe('Sourcebooks login', function() {
 
     it('Should display validation for empty user after attempted loggin', function () {
         
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')

```

Comment:
This line is lost and because of that this test case no longer working as expected